frontend/public/components/utils/log-window.jsx

```diff
@@ -151,7 +155,12 @@ export const LogWindow = withTranslation()(LogWindowWithTranslation);
 LogWindow.propTypes = {
   bufferFull: PropTypes.bool.isRequired,
   lines: PropTypes.array.isRequired,
+  wrapLines: PropTypes.bool,
   linesBehind: PropTypes.number.isRequired,
   status: PropTypes.string.isRequired,
   updateStatus: PropTypes.func.isRequired,
 };
+
+LogWindow.defaultProps = {
+  wrapLines: true,
```

Comment:
The existing behavior does not wrap, so I think we'd want to preserve that functionality for backward compatibility.

Reply:
That was my thought too, but the card says "It should probably be enabled by default to align with the Developer perspective's current behavior in Pipeline logs."  

I'll verify that in the developer view, line wrapping is on by default.  If not, I'll verify with the person who submitted the ticket

Reply:
I'm the one who wrote the ticket, and I'm just as surprised as you both about that sentence. 😄

My memory is foggy, but I must've written that after discussing it with folks on the Developer perspective side. I pinged Christian in [CONSOLE-2351](https://issues.redhat.com/browse/CONSOLE-2351) but I have a feeling that as long as the Pipeline log viewer continues to wrap by default, having all others not wrap by default should be fine.